tests/test_warc_metadata_sidecar.py

```diff
@@ -0,0 +1,144 @@
+import io
+import os
+import socket
+from logging import INFO
+from unittest.mock import patch, call
+
+import pycld2 as cld2
+from warcio.archiveiterator import ArchiveIterator
+import warc_metadata_sidecar as sidecar
+
+
+HOSTNAME = socket.gethostname()
+
+TEST_DIR = os.path.dirname(__file__)
+
+TEXT_TEST_FILE = os.path.join(TEST_DIR, 'text.warc')
+DNS_TEST_FILE = os.path.join(TEST_DIR, 'dns.warc')
+IMAGE_TEST_FILE = os.path.join(TEST_DIR, 'gif.warc')
+REVISIT_TEST_FILE = os.path.join(TEST_DIR, 'revisit.warc')
+
+MIME_TITLE = 'Identified-Payload-Type:'
+METHOD_TITLE = 'Payload-Type-Retrieval-Method:'
+RETRIEVED_TITLE = 'Payload-Type-Retrieved-By:'
+PUID_TITLE = 'Preservation-Identifier:'
```

Comment:
Rather than redefining these, I think we could use the constants set in warc_metadata_sidecar (e.g. where used in the test, use `sidecar.MIME_TITLE`).